docs/src/app/versions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
I wonder how far back this will work since pre React 0.14.0 it was `React.render` instead of `ReactDOM.render`?